applications/openshift/kubelet/kubelet_configure_event_creation/tests/ocp4/e2e.yml

```diff
@@ -1,2 +1,4 @@
 ---
-default_result: PASS
+# This rule fails by default on 4.13, and passes by default on 4.14
+default_result: FAIL
```

Comment:
Ok - good to know. We'll need to flip this bit when we start using 4.14 in CaC CI

https://github.com/openshift/release/blob/master/ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master.yaml#L12